### DISCUSSION TITLE
Welcome to hpc-social.github.io Discussions!

### DISCUSSION THREAD

**Discussion by alansill** [hpc-social/hpc-social.github.io#19] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community. For general discussions, of course, use the [hpc.social Mastodon instance itself](https://mast.hpc.social) or the [separate additional Discussions area for the Mastodon instance and policies](https://github.com/hpc-social/mastodon-instance-and-policies/discussions), but the space here is intended to allow discussions of the policies documents related to the [hpc.social web site site](https://hpc.social)  without clogging up the feed there with off-topic details and to allow deeper discussion of features we could add and other topics if needed.

**Comment by vsoch** [comment]
Hi I'm Vanessa / V / Vanessasaurus / vsoch  :wave: 

I'm here because it's important for us to foster a welcoming, fun, and diverse community, and I think I can help. I have a love for a wide range of different technologies and platforms, and to my vision is to be able to more easily work across them. I don't think I am wanting a "hybrid" approach of jumping from one to another, but rather a converged approach - the same technologies being adopted by both sides, and both sides (e.g., cloud native and HPC as examples) working together toward common goals. I would also like the focus of conferences and events to be less on "we have the biggest, baddest machines!" and more on the people, and having fun. To whatever degree I can help with that, I'm here for it!

**Comment by alansill** [comment]
I will say "same for me!" to Vanessa's comments and would like to see discussion here and on [the mast.hpc.social Mastodon instance](https://mast.hpc.social) or on Mastodon in general with the #HPC tag for ways to improve the server and on HPC and related fields in general. People and their mutual interactions are just as important as the machines to advance science, technology, and humanity and I hope we will see lots of discussions there, and here for ideas on how to improve this service.

**Comment by jwender** [comment]
Just to say Hi as well :-) The idea of having an online home for the HPC community resonates with me very much (if you can say so, I'm not a native speaker, being German and living in Germany). On Twitter I found it always somewhat dispersed and you needed the right follow list to see the posts. That is something I like with Mastodon (local timeline gives the HPC-related toots) (even if I think the term "toot" is somewhat silly (but not as silly as the German version "trööt")). 
Anyway, I am happy to help with anything.

**Comment by hestes835** [comment]
Hello everyone!! :)
I have been looking for online HPC communities for a while. Sadly the only thing I found active has been the subreddit which seems unlikely to expand outside of that space. At SC22, the student volunteers and I created a Discord with a steady pace of new people since its creation. I was thrilled to find HPC.social! I hope we have some real-time chat room set up at some point ( I have already seen talks about Matrix here :eyes: ). I am still exploring the general community here, but I have joined the mastodon and am considering joining the community blogs. 

I am an undergraduate student who started research in HPC about a year ago so I may have less insight than some people here, but if there is any way I can help, please let me know!

**Reply by vsoch** [reply]
Nice to meet you @Qeolvdxu ! We have a slack that a few of us have joined, and would love if you wanted to share the invite link with your Discord community. I'll send you an invite link to the email I found on your website!

**Comment by alansill** [comment]
Just adding a +1 to @vsoch's response. We are investigating adding Discourse and possibly also matrix.org-based applications to the hpc.social portfolio, and how to make them as well as Slack bring value. There are cost as well as maintenance, monitoring, curation, and code of conduct issues as with any online community media. WIth sufficient energy we can overcome the other issues and we are all happy to welcome you into the discussions. The ones here on GitHub provide a quick entry point, but for extended and more broad-ranging discussions, join in on the Slack instance and let us know your opinions. We also have some overall discussion going on here: https://github.com/hpc-social/hpc-social.github.io/discussions/52

Thanks and we'll look for your input as above!
